metadata.rb

```diff
@@ -10,9 +10,9 @@
 
 depends          'base'
 depends          'build-essential'
-depends          'git', '~> 4.3'
+depends          'git'
 depends          'java'
-depends          'jenkins', '~> 2.4.0'
+depends          'jenkins', '~> 5.0.0'
 depends          'osl-haproxy'
 depends          'ssh-keys'
```

Comment:
We pin down versions in environments, IIRC. I'm not sure if the cookbook uploader automatically detects situations where dependences are updated or not, but the uploder's code knows.